mne_connectivity/base.py

```diff
@@ -0,0 +1,211 @@
+import numpy as np
+
+from mne.utils import sizeof_fmt, object_size
+
+
+class _Connectivity():
+    def __init__(self, data, names, method, indices):
+        """Base class container for connectivity data.
+
+        Connectivity data is anything represents "connections"
+        between nodes as a (N, N) array. It can be symmetric, or
```

Comment:
```suggestion
        Connectivity data is anything that represents "connections"
        between nodes as a (N, N) array. It can be symmetric, or
```